### DISCUSSION TITLE
What's the state of SD card support in the pico-arduino world?

### DISCUSSION THREAD

**Discussion by madskjeldgaard** [pschatzmann/pico-arduino#21] [Q&A]
Hi

I am struggling to get an SD card to work with my Pico Arduino setup. 

I was wondering if that part of the library is mature and if things like the following still needs to be done? 

I found this in one of the readme's for the sd card examples:
```
When I first tested the functionality, I wondered why the functionality did not work properly.

It turned out that USE_SIMPLE_LITTLE_ENDIAN must not be set to 1. This is changed in the SdFatConfig.h
I also needed to lower the max speed from 16MHz to 12MHz.
```

Is that still necessary?

**Comment by pschatzmann** [comment]
I don't know since I am not actively using this library any more.

I suggest that you use Eale Phil Howers core instead....
